src/auto-evo/RunResults.cs

```diff
@@ -518,6 +523,16 @@ public List<Species> GetNewSpecies()
             return results[species].EnergyResults;
         }
 
+        public Species GetMutationResults(Species species)
+        {
+            return results[species].MutatedProperties;
+        }
+
+        public (Species SplitOff, Species SplitFrom, List<Patch> SplitOffPatches) GetSplitResults(Species species)
```

Comment:
Here's the other two

Reply:
Related to https://github.com/Revolutionary-Games/Thrive/pull/2920#discussion_r794342880. The latter is in a few lines down: https://github.com/Revolutionary-Games/Thrive/blob/4eb6e8fd5704d1098d8ba8317ea6fc8a9b510763/src/auto-evo/RunResults.cs#L932

Reply:
Why don't these actually use the full species result objects like done in other methods in this class? That's why these "get this one specific bit of the results" aren't needed for everything.

Reply:
Yeah, I guess it's too specific of a use case now. I think I'll clean these up.